source/tutorial/react-native.txt

```diff
@@ -15,7 +15,8 @@ React Native Tutorial
    :values: javascript/typescript
  
 .. meta::
-   :keywords: realm/react
+   :keywords: realm/react, code example
+   :description: Learn how to build, modify, and run a demo React Native application using Atlas Device SDK for React Native and Atlas Device Sync.
 
 Overview
```

Comment:
While we're updating these pages, can we remove all the "Overview" headings? They're in a handful of these tutorials. The GH tracker tutorial also has the est time at the top of the page, which seems more useful than at the bottom of the intro 
```suggestion
*Estimated time to complete: 30 minutes*
```